pages/wishlist/wishlist.css

```diff
@@ -0,0 +1,26 @@
+@import url("/css/general.css");
+
+.wishlist-items {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    grid-template-rows: repeat(2, auto);
```

Comment:
I might be wrong here but I think even if you do not define the rows explicitly the browser will take care of it. So maybe try removing this line.

Reply:
yes youre right Mahesh :) I will do it